src/content/general/cluster-upgrades/index.md

```diff
@@ -224,7 +224,7 @@ To help the scheduler further with being able to correctly (re-)schedule your Po
 
 Some, but not all, cluster upgrades require nodes to be upgraded. With single master clusters, this causes a downtime of the Kubernetes API that can last a few minutes.
 
-If you are running Giant Swarm on AWS, since workload cluster release v{{% first_aws_ha_controlplane_version %}} you have the option to use [high-availability masters]({{< relref "/advanced/high-availability/control-plane" >}}) instead. This will keep the Kubernetes API available even during an upgrade where nodes are rolled.
+If you are running Giant Swarm on AWS, since workload cluster release v{{% first_aws_ha_controlplane_version %}} you have the option to use a [high-availability masters]({{< relref "/advanced/high-availability/control-plane" >}}) instead. This will keep the Kubernetes API available even during an upgrade where nodes are rolled.
```

Comment:
Here it still says `high availability masters`. Shouldn't it be `highly available control plane`?

Reply:
Indeed

Reply:
```suggestion
If you are running Giant Swarm on AWS, since workload cluster release v{{% first_aws_ha_controlplane_version %}} you have the option to use a [high-availability control plane]({{< relref "/advanced/high-availability/control-plane" >}}) instead. This will keep the Kubernetes API available even during an upgrade where nodes are rolled.
```